Modules/Devices/src/Devices.Domain/Entities/Identities/IdentityDeletionProcess.cs

```diff
@@ -82,6 +82,11 @@ public bool IsActive()
         return Status is DeletionProcessStatus.Approved or DeletionProcessStatus.WaitingForApproval;
     }
 
+    public DateTime GetEndOfApprovalPeriod()
```

Comment:
A small test for this method would be good. Just to make sure nobody f**ks it up. :)